docs/netbsd.md

```diff
@@ -100,3 +131,6 @@ If something does not work, add `-debug` flag to `syz-manager`.
 - KASAN for NetBSD would be useful.
 - On Linux we have emission of exernal networking/USB traffic into kernel using tun/gadgetfs. Implementing these for NetBSD could uncover a number of high-profile bugs.
 - Last but not least, we need to support NetBSD in `syz-ci` command (including building kernel/image continuously from git).
+
+
+password protect things
```

Comment:
This is not treated as part of the list.
But do we actually want to password-protect things? This is 100% test image/setup, not having passwords can only make things simpler.